pkg/kev/envs.go

```diff
@@ -197,7 +198,7 @@ func (e *Environment) loadOverride() (*Environment, error) {
 		envVarsFromNilToBlankInService(s)
 		serviceConfig, err := newServiceConfig(s)
 		if err != nil {
-			return nil, errors.Wrapf(err, "Cannot load environment [%s], service [%s]", e.Name, name)
+			log.Debugf("cannot load environment [%s], service [%s]: err %s", e.Name, name, err.Error())
```

Comment:
I guess we'd still want to return an error here?

Reply:
no, this one is specifically caused by https://github.com/appvia/kev/blob/efece0d1e0afb7af8adea41fa32d7414d4556e91/pkg/kev/services.go#L32
which is going to be removed at some point

Reply:
OK, but if there is a service config error does the above still make sense? or should we just return immediately with the first error that occurred?

Reply:
for now we move on because we don't care about labels anymore, if it's wrong is because it is either not a label we have ported and in which case it doesn't matter or is one we ported in which case we don't care either so we move on :)